k8s/overlays/develop/.gitignore

```diff
@@ -0,0 +1 @@
+secrets.yaml
```

Comment:
do you want this ignored? we commit our secrets as they are normally encrypted

Reply:
The encrypted secrets are output to a `./k8s/overlays/develop/secrets.enc.yaml` file. I suppose I could overwrite in place instead.